app/main/views/register.py

```diff
@@ -34,69 +21,6 @@ def register():
     abort(404)
 
 
-@main.route("/register-from-org-invite", methods=["GET", "POST"])
-# TODO This is deprecated, we are now handling invites in the
-# login.gov workflow.  Leaving it here until we write the new
-# org registration.
-def register_from_org_invite():
-    invited_org_user = InvitedOrgUser.from_session()
-    if not invited_org_user:
-        abort(404, "No invited_org_user")
-
-    form = RegisterUserFromOrgInviteForm(
-        invited_org_user,
-    )
-    form.auth_type.data = "sms_auth"
-
-    if form.validate_on_submit():
-        if (
-            form.organization.data != invited_org_user.organization
-            or form.email_address.data != invited_org_user.email_address
-        ):
-            abort(400, "organization or email address doesn't match")
-        _do_registration(
-            form,
-            send_email=False,
-            send_sms=True,
-            organization_id=invited_org_user.organization,
-        )
-        invited_org_user.accept_invite()
-
-        return redirect(url_for("main.verify"))
-    return render_template(
-        "views/register-from-org-invite.html",
-        invited_org_user=invited_org_user,
-        form=form,
-    )
-
-
-def _do_registration(form, send_sms=True, send_email=True, organization_id=None):
-    user = User.from_email_address_or_none(form.email_address.data)
-    if user:
-        if send_email:
-            user.send_already_registered_email()
-        session["expiry_date"] = str(datetime.utcnow() + timedelta(hours=1))
-        session["user_details"] = {"email": user.email_address, "id": user.id}
-    else:
-        user = User.register(
-            name=form.name.data,
-            email_address=form.email_address.data,
-            mobile_number=form.mobile_number.data,
-            password=form.password.data,
-            auth_type=form.auth_type.data,
-        )
-
-        if send_email:
-            user.send_verify_email()
-
-        if send_sms:
-            user.send_verify_code()
-        session["expiry_date"] = str(datetime.utcnow() + timedelta(hours=1))
-        session["user_details"] = {"email": user.email_address, "id": user.id}
-    if organization_id:
-        session["organization_id"] = organization_id
-
-
```

Comment:
Oh, I think that the comment at the top answers my questions above, but want to be sure and double check!